tools/parquetGen/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+include_directories(${ENGINE_COMMON_INCLUDES})
```

Comment:
We probably need to rename the directory so that it is in line with the resulting binary name.